assets/js/theme/common/product-details.js

```diff
@@ -216,8 +216,14 @@ export default class ProductDetails {
                 $container: $('.form-field--stock', $scope),
                 $input: $('[data-product-stock]', $scope),
             },
-            $sku: $('[data-product-sku]'),
-            $upc: $('[data-product-upc]'),
+            sku: {
+                $dt: $('dt.sku-label', $scope),
+                $dd: $('[data-product-sku]', $scope),
```

Comment:
can we just name the keys as `label` and `value` so it is easier to read.

Reply:
yea I suppose so - was just following the format already in place